src/Elcodi/Component/Geo/Command/LocationPopulateCommand.php

```diff
@@ -82,28 +95,39 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $country = $input->getArgument('country');
-        $reloadSource = $input->hasOption('reload-source');
+        $countryCodes = $input->getArgument('country');
+        $countryCodes = explode(',', $countryCodes);
+
         $formatter = $output->getFormatter();
-        $formatter->setStyle('header', new OutputFormatterStyle(
-            'green', null, []
-        ));
-        $formatter->setStyle('body', new OutputFormatterStyle(
-            'white', null, []
-        ));
+        $formatter->setStyle('header', new OutputFormatterStyle('green'));
+        $formatter->setStyle('body', new OutputFormatterStyle('white'));
 
         $output->writeln('');
-        $output->writeln('<header>[Geo]</header> <body>Populating your database with '.$country.' fixtures</body>');
         $output->writeln('<header>[Geo]</header> <body>This process may take a few minutes. Please, be patient</body>');
-        $output->writeln('<header>[Geo]</header> <body>Process started...</body>');
 
-        $this
-            ->geoPopulator
-            ->populateCountries(
-                $output,
-                [$country],
-                $reloadSource
-            );
+        foreach ($countryCodes as $countryCode) {
+            $output->writeln('<header>[Geo]</header> <body>Populating country code: '.$countryCode);
+
+            $locations = $this
+                ->locationPopulator
+                ->populate(
+                    $countryCode,
+                    $output
+                );
+
+            $output->writeln('<header>[Geo]</header> <body>Flushing manager started</body>');
+
+            $start = new DateTime();
+
+            $this
+                ->locationDirector
+                ->save($locations);
+
+            $finish = new DateTime();
+            $elapsed = $finish->diff($start);
```

Comment:
Same as https://github.com/elcodi/elcodi/pull/517#commitcomment-9760766.
We should use `$elapsed = $finish->getTimestamp() - $start->getTimestamp();` or `$elapsed = $finish->format('U') - $start->format('U');`.